### PR TITLE
Aero Tray 1.0.1: Fix injection stuff

### PR DESCRIPTION
**Pull request by aubymori**
This mod needs to hook into `winlogon.exe` to successfully capture Explorer starting.

### CODE REVIEW

File: mods/aero-tray.wh.cpp [hunk 1]
```diff
@@ -16,6 +17,11 @@
 This mod restores the tray overflow from Windows 7 and 8, including the "Customize..." link,
 drawing tray icons with the system theme, and the general appearance.
 
+# IMPORTANT: READ!
+This mod needs to hook into `winlogon.exe` to successfully capture Explorer starting. Please
+navigate to Windhawk's Settings, Advanced settings, More advanced settings, and make sure that
+`winlogon.exe` is in the Process inclusion list.
```
Comment (m417z) [inline mods/aero-tray.wh.cpp:22]: You don't need to `@include winlogon.exe` for this, it's enough to add it to the Process inclusion list.

Comment (m417z) [reply mods/aero-tray.wh.cpp:22]: For more technical details see the blog post:
https://m417z.com/Implementing-Global-Injection-and-Hooking-in-Windows/
And the high level architecture:
https://github.com/ramensoftware/windhawk?tab=readme-ov-file#technical-details

You want Windhawk to be injected into `winlogon.exe` for this:
image

But loading your mod which does nothing has no value.

**Comment by aubymori** [comment]
@m417z should be good now

File: mods/aero-tray.wh.cpp [hunk 2]
```diff
@@ -946,12 +950,21 @@ LPCWSTR SIB_PATH = L"%PROGRAMFILES(X86)%\\StartIsBack\\StartIsBack64.dll";
...
 BOOL Wh_ModInit(void)
 {
+    WCHAR szPath[MAX_PATH];
+    GetModuleFileNameW(GetModuleHandleW(NULL), szPath, MAX_PATH);
+    LPWSTR pszBackslash = wcsrchr(szPath, L'\\');
+    if (0 != wcsicmp(pszBackslash + 1, L"explorer.exe"))
+    {
+        Wh_Log(L"Not Explorer");
+        return FALSE;
+    }
```
Comment (m417z) [inline mods/aero-tray.wh.cpp]: You should no longer need this check now.

File: mods/aero-tray.wh.cpp [hunk 3]
```diff
@@ -16,6 +16,11 @@
 This mod restores the tray overflow from Windows 7 and 8, including the "Customize..." link,
 drawing tray icons with the system theme, and the general appearance.
 
+# IMPORTANT: READ!
+This mod needs to hook into `winlogon.exe` to successfully capture Explorer starting. Please
```
Comment (m417z) [inline mods/aero-tray.wh.cpp]: To be more precise, it should say "Windhawk needs to hook into `winlogon.exe`...".

**Pull request merged**